Mage/src/main/java/mage/players/Player.java

```diff
@@ -209,6 +209,8 @@ default boolean isComputer() {
 
     Cards getHand();
 
+    void incrementLandsPlayed();
```

Comment:
There are other type of changes for the field:
`this.landsPlayed = 0;`

So must use for same code style:
* landsPlayedInc()
* landsPlayedReset()